src/cli/data.py

```diff
@@ -586,9 +594,14 @@ def delete(self):
         """Delete rows, columns, databases and/or folders"""
 
         if self.app.pargs.columns:
+            if self.app.pargs.database is None:
+                raise DeepOriginException(
+                    "You must specify a database to delete columns in using --database"
```

Comment:
I like to provide instructions first and explanation second. How about this?

> Use the --database argument to specify the parent database. To delete a column, the parent database must be specified.

Reply:
fixed